File: android/CustomListView/app/src/main/java/com/example/customlistview/MainActivity.java
package com.example.customlistview;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

// 커스텀 리스트뷰
// 1. 앱 화면 그려보기
// 2. 아이템 레이아웃 구성하기
// 3. 커스텀 어댑터 구현 (BaseAdapter 상속)
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

    }
}

File: android/radiobar/app/src/main/java/com/example/radiobar/MainActivity.kt
package com.example.radiobar

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        radioGroup.setOnCheckedChangeListener { radioGroup, i ->
            when(i) {
                R.id.radio_apple -> textView.text = "${radio_apple.text}"
                R.id.radio_banana -> textView.text = "${radio_banana.text}"
                R.id.radio_orenge -> textView.text = "${radio_orenge.text}"
            }
        }
    }
}